bcda/models/models.go

```diff
@@ -109,15 +109,15 @@ func (job *Job) GetEnqueJobs(t string) (enqueJobs []*que.Job, err error) {
 		return nil, err
 	}
 
-	beneficiaryIDs, err := aco.GetBeneficiaryIDs()
+	beneficiaries, err := aco.GetBeneficiaries(false)
```

Comment:
Maybe a comment to indicate why we're passing `false`?

Reply:
✅